crm_project_task/wizards/crm_create_task.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class CrmCreateTAsk(models.TransientModel):
+    _name = "crm.create.task"
+    _description = "Wizard to create task"
+
+    lead_id = fields.Many2one("crm.lead")
+    name_task = fields.Char()
+    description = fields.Html()
+
+    def create_task(self):
+        project = self.env.company.crm_default_project_id
+        if not project:
+            raise UserError(
+                _(
+                    "Project not configured in settings,"
+                    "please contact with your administrator."
+                )
+            )
+        task = (
+            self.env["project.task"]
+            .sudo()
+            .create(
+                {
+                    "name": self.name_task,
+                    "project_id": project.id,
+                    "partner_id": self.lead_id.partner_id.id,
+                    "description": self.description,
+                    "user_ids": [(6, 0, [])],
+                }
+            )
+        )
+        task.message_post(
+            body=_(
+                "Task created from lead/opportunity "
+                "<a href=# data-oe-model=crm.lead data-oe-id=%(lead)d>%(name)s</a>.",
+                lead=self.lead_id,
+                name=self.lead_id.name,
+            )
+        )
+        self.lead_id.message_post(
+            body=_(
+                "Task <a href=# data-oe-model=project.task "
+                "data-oe-id=%(task)d>%(name)s</a> created.",
+                task=task,
+                name=task.display_name,
+            )
+        )
```

Comment:
When you finish, you should return a client action that directs the user to the created task directly. They should be able to still go back to the lead using breadcrumbs if they want.

Reply:
@yajo , the task is not for the salesman, and in fact the salesman don't have access to project app